packages/electron-updater/src/AppUpdater.ts

```diff
@@ -86,6 +86,12 @@ export abstract class AppUpdater extends (EventEmitter as new () => TypedEmitter
 
   disableWebInstaller = false
 
+  /**
+   * Install dir for Nsis Updater
+   *
+   */
+  installDir = ''
```

Comment:
I'd recommend using `installDirectory: string | null = null` since that will be picked up during doc generation